web/pages/index/+Page.tsx

```diff
@@ -61,6 +61,7 @@ const INTEGRATIONS: IntegrationItem[] = [
     { name: 'DevDocs', type: 'provider', slug: 'devdocs' },
     { name: 'Confluence', type: 'provider', slug: 'confluence' },
     { name: 'Jira', type: 'provider', slug: 'jira' },
+    { name: 'Azure DevOps Work Items', type: 'provider', slug: 'azure-devops-workitems' },
```

Comment:
you need a corresponding page entry. eg see web/content/docs/providers/devdocs.mdx